app/apps/client/orchestrator.js

```diff
@@ -65,12 +65,12 @@ class AppClientOrchestrator {
 
 	getAppsFromMarketplace = async () => {
 		const appsOverviews = await APIClient.get('apps', { marketplace: 'true' });
-		return appsOverviews.map(({ latest, price, pricingPlans, purchaseType, permissions }) => ({
+		return appsOverviews.map(({ latest, price, pricingPlans, purchaseType }) => ({
 			...latest,
 			price,
 			pricingPlans,
 			purchaseType,
-			permissions,
+			permissions: latest.permissions,
```

Comment:
isn't just a matter of removing this? since there is a spread of `latest` that will populate `permissions` already:
```suggestion

```

Reply:
The idea fix is for someone who is more knowledgeable of the UI to pull the permissions from the latest property. This was my first time interacting with the new UI and so this is what I ended up with.

Reply:
I was just talking about the code actually.. since these both produce the same result, I'd recommend doing the latter:

this is the proposed on this PR:
```javascript
		return appsOverviews.map(({ latest, price, pricingPlans, purchaseType }) => ({
			...latest,
			price,
			pricingPlans,
			purchaseType,
			permissions: latest.permissions,
		}));
```

and this would be my suggestion:

```javascript
		return appsOverviews.map(({ latest, price, pricingPlans, purchaseType }) => ({
			...latest,
			price,
			pricingPlans,
			purchaseType,
		}));
```

Reply:
It makes sense. I'll give it another testing round to make sure :)